common/src/chain/transaction/signature/mod.rs

```diff
@@ -108,64 +109,22 @@ pub trait Signable {
 }
 
 pub trait Transactable: Signable {
-    fn signatures(&self) -> Option<&[InputWitness]>;
+    fn signatures(&self) -> Vec<Option<InputWitness>>;
```

Comment:
This is still a change I don't like. Do you still need this after having introduced mintscript? Can you please revert all the changes in this file?

Reply:
The reason I did this is because I want `PartiallySignedTransaction` to implement this trait and unify signature verification.